dut/drivers.wake

```diff
@@ -0,0 +1,109 @@
+
+tuple BasicDriverPlan =
+  global DuhFile:             Path
+  global OMFile:              Unit => Path
```

Comment:
If you get the OMFile from building the RTL, you need to delay execution.

Reply:
I'm still confused about why we need to delay the execution here. Couldn't you just delay the construction of the BasicDriverPlan until after OM is constructed?

Reply:
also I don't think the tuple that we publish to `basicDriverPlans` should be the same tuple that is used as the plan for `makeBasicDrivers`. because like you said the OMFile isn't available until after the dut is elaborated same with things like the `BSPDirectory` and `DUTName`. There are also options that don't really make sense in the context of your driver like `OverwriteExisting` or `BaseHeader`.